addons/website_sale/security/ir.model.access.csv

```diff
@@ -53,7 +53,3 @@ access_ecom_extra_fields_public_public,access_ecom_extra_field public,model_webs
 access_ecom_extra_fields_public_portal,access_ecom_extra_field public,model_website_sale_extra_field,base.group_portal,1,0,0,0
 access_ecom_extra_fields_public_employee,access_ecom_extra_field public,model_website_sale_extra_field,base.group_user,1,0,0,0
 access_ecom_extra_fields_restricted_editor,access_ecom_extra_field restricted_editor,model_website_sale_extra_field,website.group_website_restricted_editor,1,1,1,1
-access_website_base_unit_public_public,website.base.unit public,model_website_base_unit,base.group_public,1,0,0,0
-access_website_base_unit_public_portal,website.base.unit public,model_website_base_unit,base.group_portal,1,0,0,0
-access_website_base_unit_public_employee,website.base.unit public,model_website_base_unit,base.group_user,1,0,0,0
-access_website_base_unit_sale_manager,website.base.unit sale manager,model_website_base_unit,sales_team.group_sale_manager,1,1,1,1
```

Comment:
Sales manager should keep their access to the moved model.
Being a sales manager doesn't imply being a product manager, I believe this rule should be somehow kept.
Same for the portal and public access rules.

Reply:
Moved the sales manager to `sales` and the others to `product` itself.